Makefile

```diff
@@ -23,3 +23,14 @@ cmddocs:
 ITER8_BIN ?= /usr/local/bin/iter8
 build:
 	cd clibase && go build -o $(ITER8_BIN)
+
+
+ITER8_IMG ?= iter8/iter8cli:latest
+docker-build:
+	docker build -f Dockerfile.k8s -t $(ITER8_IMG) .
+
+docker-push:
+	docker push $(ITER8_IMG)
+
+build-k8s:
```

Comment:
I believe we want only one build. Whatever is called `build-k8s` should be the default `build`.

If at all we need `make build-basecli`, we will do that eventually, but I don't think this should be there at all (in other words, we should remove the main from basecli).